src/main/kotlin/kweb/html/Document.kt

```diff
@@ -39,10 +39,10 @@ class Document(val receiver: WebBrowser) : EventGenerator<Document> {
         return head
     }
 
-    val origin = receiver.evaluate("document.origin")
+    val origin = receiver.callJsFunctionWithResult("return document.origin")
 
     fun execCommand(command: String) {
-        receiver.execute("document.execCommand(\"$command\");")
+        receiver.callJsFunction("document.execCommand({});", command)
```

Comment:
I think the command here will need to be in quotes for this to work.

Reply:
Actually, this might be one situation where parameterizing the function is more trouble than it's worth, because if we surround command with quotes then it won't be treated as a variable by JavaScript.

Reply:
Another option would be to remove this function completely because it will almost always be better to use callJs().

Reply:
The {} becomes a variable name in javascript. This means that execCommand({}), will become execCommand(generated_var_name). The command string will then be set as the value of generated_var_name. This means we do not need to put quotes around the braces. It will work as is.

That being said, you are right that the function may be redundant, and could be removed.